src/react-components/src/components/03-organisms/Hero/Hero.tsx

```diff
@@ -0,0 +1,115 @@
+// OH-30 Header with Image Right
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Image from "../../01-atoms/Images/Image/Image";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import { HeroTypes } from "./HeroTypes";
+
+export interface HeroProps {
+  /** Can be Primary, secondary, tertiary, or 50/50. */
+  heroType?: HeroTypes;
+
+  /** Required ID for tabfocus. */
+  heroId: string;
+
+  /** Required heading element. */
+  heading: JSX.Element;
+
+   /** Optional subheader that displays underneath the
+    * required heading element.
+    */
+  subHeaderText?: string;
```

Comment:
change this to JSX.Element

Reply:
This is my bad originally, I thought strings were good enough to do this with, but if you, say, ever have a subtitle with a link in it, or a <b></b> in it, it wouldn't be possible, and it doesn't make sense to go to a PM and say "I can't make part of the subheader italicized"

Reply:
That's true. On the larger scale of this, it should be a written down requirement that this _can_ be italicized. Otherwise, you'd be coding for possible scenarios that might not be implemented.

Reply:
I've updated this to use `<BodyText>`.